bbcode-src/preset.js

```diff
@@ -4,7 +4,7 @@ import { bg } from "./tags/background";
 import { block } from "./tags/block";
 import { blockquote } from "./tags/blockquote";
 import { border } from "./tags/border";
-import { centerblock } from "/tags/centerblock";
+import { centerblock } from "./tags/centerblock";
```

Comment:
@hartleyk1 @Alteras1  This period was causing issues with my local dev. Didn't check on T, but updated it so it wouldn't keep throwing errors on plugin build.